app/scripts/ui/basic-button.js

```diff
@@ -84,12 +100,16 @@ function BasicButton(options, ui, drawingTool) {
   }
 }
 
-BasicButton.prototype.setLabel = function (label) {
-  this.$label.text(label);
+BasicButton.prototype.setLabelOrIcon = function (tool) {
+  if (tool.icon && this.$icon) {
+    this.$icon.attr('src', tool.icon);
+  } else {
+    this.$label.text(tool.label);
+  }
 };
 
 BasicButton.prototype.getLabel = function () {
-  return this.$label.text();
+  // return this.$label.text();
```

Comment:
Is that intended? Shouldn't we return something here?

Reply:
Good catch.  I need to remove the old label code now that the icon font is not used.  I will do that before I mark the PR ready for review.